lib/smart_proxy_pulp_plugin/disk_usage.rb

```diff
@@ -0,0 +1,56 @@
+module PulpProxy
+  class DiskUsage
+    include ::Proxy::Util
+    SIZE = { :kilobyte => 1_024, :megabyte => 1_048_576, :gigabyte => 1_073_741_824, :terabyte => 1_099_511_627_776 }
+
+    attr_reader :path, :stat, :size
+
+    def initialize(opts ={})
+      raise(::Proxy::Error::ConfigurationError, 'Unable to continue - must provide a path.') if opts[:path].nil?
+      @paths_hash = path_hash(opts[:path])
+      @path = @paths_hash.values
+      @size = SIZE[opts[:size]] || SIZE[:gigabyte]
```

Comment:
Please use kilobyte as the sane default value, because the default output is useless:

    {"pulp_dir":{"filesystem":"tmpfs","1g-blocks":"8","used":"1","available":"8","percent":"1%","mounted":"/tmp","path":"/tmp","size":"gigabyte"}}